Amazon.IonObjectMapper/Serializer/Primitive/IonClobSerializer.cs

```diff
@@ -21,6 +22,31 @@ namespace Amazon.IonObjectMapper
     /// </summary>
     public class IonClobSerializer : IonSerializer<string>
     {
+        private readonly Encoding encoding;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="IonGuidSerializer"/> class.
+        /// </summary>
+        ///
+        /// <param name="encoding">Type of encoding for Clob.</param>
+        public IonClobSerializer(string encodingString = "Unicode")
```

Comment:
The reason `IonClobSerializer` takes a `string` rather than a `Encoding` is because in C#, [`Only primitive constants or arrays of primitives can be used as attribute parameters`](https://stackoverflow.com/questions/3192833/why-decimal-is-not-a-valid-attribute-parameter-type)

Reply:
could we consider to use `enum` instead of `string`? and enum supports [switch expression](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/switch-expression)